src/app/content/highlights/hooks/locationChange.ts

```diff
@@ -41,12 +40,11 @@ const hookBody = (services: MiddlewareAPI & AppServices) => async(action?: AnyAc
       pagination: {page: 1, sourceIds: [page.id], perPage: maxHighlightsApiPageSize},
     });
   } catch (error) {
-    Sentry.captureException(error);
-
-    if (action && action.type !== getType(receivePageFocus)) {
-      dispatch(addToast(toastMessageKeys.higlights.failure.load, {destination: 'page', shouldAutoDismiss: false}));
+    if (action && action.type !== getType(receivePageFocus) && !(error instanceof CustomApplicationError)) {
+      throw new HighlightLoadError({ destination: 'page', shouldAutoDismiss: false });
     }
-    return;
+
+    throw error;
```

Comment:
because of the extra conditions you don't necessarily know this is a `CustomApplicationError` at this point

Reply:
We can throw it even if this isn't  `CustomApplicationError`